src/Grpc/GrpcClientStream.php

```diff
@@ -84,12 +70,7 @@ public function write($request)
      */
     public function readResponse()
     {
-        list($response, $status) = $this->call->wait();
-        if ($status->code == Grpc\STATUS_OK) {
-            return $response;
-        } else {
-            throw ApiException::createFromStdClass($status);
-        }
+        return $this->call->wait();
```

Comment:
This is a bit academic since there are no examples of client streaming in any APIs, but I don't think this is correct. The `$this->call` object is still a [ClientStreamingCall](https://github.com/grpc/grpc-php/blob/master/src/lib/ClientStreamingCall.php) object, so `wait` will still return `[$response, $status]`.  We should reuse the implementation of checking the status and throwing rather than repeating it though.

Reply:
I think this happened because I was mocking the call wrong. Will look into it and fix.